src/server/pkg/worker/master.go

```diff
@@ -38,8 +43,21 @@ const (
 	masterLockPath = "_master_worker_lock"
 )
 
+func (a *APIServer) getMasterLogger() *taggedLogger {
```

Comment:
Yes, this allows us to do `pachtctl get-logs --job ... --master` to see what's going on with the master. Works even if the master has switched between pods.